### PR TITLE
Update SELL info considering ExchangeSDK release

### PR DESCRIPTION
**Pull request by sprohaszka-ledger**
[LIVE-9024]

[LIVE-9024]: https://ledgerhq.atlassian.net/browse/LIVE-9024?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: pages/docs/exchange/sell/providers-liveapp.mdx [hunk 1]
```diff
@@ -22,7 +22,34 @@ You will need to set up the SDK with your `providerId` attributed upon request b
...
+* `accountId`: user's account id, [previously given to the LiveApp](./providers-liveapp) as a query parameter
+* `amount`: amount that will be sell
+* `feeStrategy`: `SLOW`, `MEDIUM`, `FAST` or `CUSTOM`.
+* `getSellPayload`: callback method called by the sell process to retrieve the payload (see below)
+
+The `getSellPayload` is a callback that during the Sell process will be called with a `nonce`.
+This value is required to craft on your side the Sell Payload, which will be returned by the callback.
+
+```js copy
+  exchangeSDK.swap({
```
Comment (cpellizzari-ledger) [inline pages/docs/exchange/sell/providers-liveapp.mdx:37]: exchangeSDK.sell maybe ?

File: pages/docs/exchange/sell/providers-liveapp.mdx [hunk 2]
```diff
@@ -22,7 +22,34 @@ You will need to set up the SDK with your `providerId` attributed upon request b
 
 ### SELL method
 
-[TBD] Contact Ledger for more info.
+This method will need you to provide the following parameters:
+
+* `quoteId`: the id coming from your system and [forwarded to your LiveApp](./providers-liveapp) as a query parameter
+* `accountId`: user's account id, [previously given to the LiveApp](./providers-liveapp) as a query parameter
+* `amount`: amount that will be sell
```
Comment (cpellizzari-ledger) [inline pages/docs/exchange/sell/providers-liveapp.mdx:29]: that will be sold

File: pages/docs/exchange/sell/providers-liveapp.mdx [hunk 3]
```diff
@@ -22,7 +22,34 @@ You will need to set up the SDK with your `providerId` attributed upon request b
...
+  exchangeSDK.swap({
+    quoteId: "1234",
+    accountId: "07AB5930-C73A-433F-A2FA-920640AF3A02",
+    amount: "0.8",
+    feeStrategy: "SLOW",
+    getSellPayload: (nonce, sellAddress, amount) => {
+      // Crafting Payload...
+      return {
+        recipientAddress: "0xc0ffee254729296a45a3885639AC7E10F9d54979"
+        amount: BigInt("800000000000000000")
```
Comment (cpellizzari-ledger) [inline pages/docs/exchange/sell/providers-liveapp.mdx:46]: we should document when the amount is expected in smallestDenomination and when it's in natural unit.

Comment (cpellizzari-ledger) [reply pages/docs/exchange/sell/providers-liveapp.mdx:46]: maybe better naming of the vars : amountInSmallestDenomination / amountInUsualUnit ?

File: pages/docs/exchange/sell/providers-liveapp.mdx [hunk 4]
```diff
@@ -22,7 +22,34 @@ You will need to set up the SDK with your `providerId` attributed upon request b
 
 ### SELL method
 
-[TBD] Contact Ledger for more info.
+This method will need you to provide the following parameters:
+
+* `quoteId`: the id coming from your system and [forwarded to your LiveApp](./providers-liveapp) as a query parameter
+* `accountId`: user's account id, [previously given to the LiveApp](./providers-liveapp) as a query parameter
+* `amount`: amount that will be sell
+* `feeStrategy`: `SLOW`, `MEDIUM`, `FAST` or `CUSTOM`.
```
[the remark this answers is not in the document]
Comment (sprohaszka-ledger) [reply pages/docs/exchange/sell/providers-liveapp.mdx:30]: The fee is handle the same way it is handled by the swap (AFAIK). This is more on Ledger Live side :/

**Review by pdesreux-ledger — APPROVED** [review]

**Pull request merged**